packages/block-editor/src/components/block-list/index.native.js

```diff
@@ -286,13 +291,15 @@ export default compose( [
 		} = select( 'core/block-editor' );
 
 		const selectedBlockClientId = getSelectedBlockClientId();
+		const blockClientIds = getBlockOrder( rootClientId );
 
 		return {
-			blockClientIds: getBlockOrder( rootClientId ),
```

Comment:
Hey @mchowning nice work here :)
Just curious about this line of the code. Why we don't need to set blockClientId's anymore with getBlockOrder( rootClientId ) ?

Reply:
We definitly still need this! My intention was just to extract the assignment of `getBlockOrder( rootClientId )` to a `blockClientIds` variable on [line 294](https://github.com/WordPress/gutenberg/blob/ecf8496fbc92f397c503014e1581475e6eb0a94b/packages/block-editor/src/components/block-list/index.native.js#L294), and then use that variable to return both the `blockClientIds` ([line 297](https://github.com/WordPress/gutenberg/blob/ecf8496fbc92f397c503014e1581475e6eb0a94b/packages/block-editor/src/components/block-list/index.native.js#L297)) and the `firstBlock` ([line 302](https://github.com/WordPress/gutenberg/blob/ecf8496fbc92f397c503014e1581475e6eb0a94b/packages/block-editor/src/components/block-list/index.native.js#L302)):

```
		const blockClientIds = getBlockOrder( rootClientId );     // line 294

		return {
			blockClientIds,                                   // line 297
                         [...]
			firstBlock: getBlock( blockClientIds[ 0 ] ),      // line 302
                         [...]
		};
```

I believe my changes should not alter the value that is being returned as the `blockClientIds`, so if I am altering that in some way, definitely let me know.

Reply:
Thanks, @mchowning, I definitely missed 294. 
It seems good to me.